mlir/test/Dialect/MIOpen/lowering_blockwise_gemm_v2.mlir

```diff
@@ -1,18 +1,24 @@
-// XFAIL: *
 // RUN: miopen-opt -miopen-lowering-step3 %s | FileCheck %s
 
-func @miopen_blockwise_gemm_v2_two_results(%matrixA : memref<12288xf32, 3>, %matrixB : memref<12288xf32, 3>) -> (vector<32xf32>, vector<32xf32>) {
+#map = affine_map<(d0) -> (d0 + 512)>
+
+func @miopen_blockwise_gemm_v2_two_results(%matrixA : memref<512xf32, 3>, %matrixB : memref<512xf32, #map, 3>, %bufferA : memref<2xvector<2xf32>, 5>, %bufferB : memref<2xvector<2xf32>, 5>) -> (vector<32xf32>, vector<32xf32>) {
```

Comment:
Re this test: I don't think this'll properly exercise the function we care about. To match how `blockwise_gemm_v2` gets invoked, you'd want to pass in a `Slice` transform_map with `#map` as its backing affine map

Reply:
The test was marked XFAIL in `miopen-dialect` for some time. It was originally created to test if a `blockwise_gemm_v2` op could be lowered to `xdlops_gemm_v2` successfully. We can improve the test, and also `lowering_xdlops_gemm_v2.mlir` to match more closely to how we really use these ops. Will do that in a separate cleanup PR.

Reply:
This test is actually already reflecting how we use `blockwise_gemm_v2`. The second memref does carry an affine map which is the result of `Slice`.